lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -699,6 +699,34 @@ def _save_console_log_and_check_panic(
             if check_serial_console is True:
                 check_panic(log_response_content.decode("utf-8"), "provision", log)
 
+    def _get_features_settings(self, node: Node) -> Dict[str, str]:
+        platform_runbook = cast(schema.Platform, self.runbook)
+        information: Dict[str, Any] = {}
+        if platform_runbook.capture_features_settings is False:
+            return information
+        if node.is_connected and node.is_posix:
```

Comment:
Why does it need the two conditions? I think feature settings can be got for any condition.